torchtune/models/phi3/_sentencepiece.py

```diff
@@ -0,0 +1,230 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+from typing import List, Optional, Tuple
+
+from sentencepiece import SentencePieceProcessor
+from torchtune.data._types import Message
+from torchtune.data._utils import truncate
+
+
+class SentencePieceTokenizer:
```

Comment:
So far we've been following the convention that model specific components make this explicit in the name so there isn't any confusion. Can we rename this to something like ```Phi3SentencePieceTokenizer```?